Dockerfile

```diff
@@ -2,13 +2,23 @@ FROM node:16-bullseye
 
 ARG REACT_APP_NRFESAMPLEAPP_VERSION
 ARG REACT_APP_SERVER_URL
+ARG REACT_APP_KC_URL
+ARG REACT_APP_KC_REALM
+ARG REACT_APP_KC_CLIENT_ID
 
 WORKDIR /app
 COPY . .
 
 ENV DISABLE_ESLINT_PLUGIN=true
 ENV REACT_APP_NRFESAMPLEAPP_VERSION=$REACT_APP_NRFESAMPLEAPP_VERSION
 ENV REACT_APP_SERVER_URL=$REACT_APP_SERVER_URL
+ENV REACT_APP_KC_URL=$REACT_APP_KC_URL
```

Comment:
Those changes will conflict with the ones on [ci(docker): changing docker release](https://github.com/bcgov/nr-frontend-starting-app/pull/29). Make sure to align both before merging